CONTRIBUTING.md

```diff
@@ -1,256 +1,339 @@
+When the pull request has been submitted, and looked over by 1 or more reviewers / code owners and it requires changes, the pull request will be marked with the label [`0 - Waiting on User`](https://github.com/chocolatey-community/chocolatey-packages/labels/0%20-%20Waiting%20on%20User) in which case you need to update the pull request again within 14 days.
+If the pull request is not updated within this time, the pull request will be automatically closed.
+If there is no need to update, but instead a response is needed then adding a comment on the threads created by the reviewer / code owner is enough.
+If you know you won't be able to update the pull request within 14 days, it may be better to temporarily close the issue to allow other users to contribute a pull request if they have time.
+**NOTE: Do not add a comment just to circumvent the automatic closure, this may lead the pull request to be closed immediately instead**
 
-### 1.1.10 Dependency versions
+## Conventions / Guidelines
 
-1. When taking a dependency **on an extension, specify the minimum version**.
-Without this, any version satisfies the dependency. That means unless someone upgrades the extension outside of their process or incidentally install some package that uses newer version explicitly set, it will not automatically upgrade to the latest version.
-1. When creating a dependency **for virtual package, use exact version** of the dependent package (.install or .portable) which should be the same as that of virtual package.
-1. When taking a dependency on **anything else, specify minimum or exact version**.
+All packages in this repository are expected to pass all Requirement, Guideline and Suggestions checks on the Chocolatey Community Repository when it is possible.
+Additionally, all packages are also expected to following the additional Conventions / Guidelines we have outlined in this section.
 
-### 1.1.11 Provide uninstaller only if needed
+### Manual or automatic
 
-Packages should have uninstaller if auto uninstall doesn't work.
+When working on packages in this repository, a decision must be made whether the package should be an automatic or manual package.
+In most cases it is preferable that a package is made automatic, as these can then be updated without doing manual changes when there is a new release of the
+software that is being packaged.
+The only time an automatic package should be considered to be a manual package is if the underlying software that is being packaged has not been updated for 3 or more years.
+New packages contributed/migrated to this repository should always be created as automatic packages.
 
-### 1.1.12 Ensure compatibility with PowerShell v2+
+All automatic packages are expected to make use of the [AU](https://github.com/chocolatey-community/chocolatey-au) module, and work in Windows PowerShell 5.x, PowerShell Core is not supported.
```

Comment:
```suggestion
All automatic packages are expected to make use of the [AU](https://github.com/chocolatey-community/chocolatey-au) module, and work in Windows PowerShell 5.x, as the builds do not run PowerShell Core.
```

Reply:
Here is the thing, it may run on PowerShell Core, but it may behave differently than we expect. So, would this change still be correct in this case?

Reply:
I guess I just wanted to highlight the _reason_ for us not allowing scripts that _may_ work in Core, so that people would understand - AppVeyor runs 5, currently, and that's what actually produces and pushes our packages. What we actually care about is that it works correctly in 5, and on our CI.

```suggestion
All automatic packages should make use of the [AU](https://github.com/chocolatey-community/chocolatey-au) module, and must work in Windows PowerShell 5.x.
```